index.js

```diff
@@ -269,25 +241,29 @@ function Trace(options) {
  * https://www.youtube.com/watch?v=NCFDqeo7AeY}
  *
  * @example
- * trace.startAgent();
+ * trace.start();
  */
-Trace.prototype.startAgent = function(config) {
-  publicAgent.startAgent(config);
+function start(config) {
+  publicAgent.start(config);
   return publicAgent;
-};
+}
 
-Trace.prototype.isActive = function() {
+function isActive() {
   return publicAgent.isActive();
-};
+}
 
-Trace.prototype.get = function() {
+function get() {
   return publicAgent.get();
-};
+}
 
 global._google_trace_agent = publicAgent;
-module.exports = Trace;
+module.exports = {
+  start: start,
+  isActive: isActive,
```

Comment:
Do we need to expose `isActive` on the public interface or can people ask the agent they receive from `start`/`get`?